src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,79 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
 public class HelloWorldTest {
 
+    private static RegistrationService registrationService;
+    private static User user;
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+        user.setId(1L);
+        user.setAge(21);
+        user.setLogin("validLogin");
+        user.setPassword("validPassword");
+    }
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new core.basesyntax.service.RegistrationServiceImpl();
+        user = new User();
+    }
+
+    @Test
+    void register_validUser_ok() {
+        User expected = user;
+        User actual = registrationService.register(user);
+        Assertions.assertEquals(expected, actual);
+    }
+
+    @Test
+    void register_userExist_notOk() {
+        registrationService.register(user);
```

Comment:
lets add user to Storage directly(as we cant be sure that register method works correctly)